.travis.yml

```diff
@@ -7,13 +7,16 @@ services:
 matrix:
   include:
     - env: TO_TEST=build
-    - env: TO_TEST=e2e
+    - env: TO_TEST=ephemeral
```

Comment:
Are all these tests using the same "oc cluster up" instance ?
travis VM has resource limits.Some of the tests can get delayed and can break because of the delay if there are more testcases.